pandas/tests/scalar/timedelta/test_construction.py

```diff
@@ -85,10 +85,6 @@ def test_construction():
     with pytest.raises(ValueError):
         Timedelta('10 days -1 h 1.5m 1s 3us')
 
-    # no units specified
-    with pytest.raises(ValueError):
-        Timedelta('3.1415')
```

Comment:
we don't allow this, pls revert. a string w/o a unit must raise.

Reply:
Well, looking back at the issue https://github.com/pandas-dev/pandas/issues/12136, there was discussion about this and said we would allow this. So that is a good reason this is PR is doing that! :-) 

(but personally, I would simply not allow it)

Reply:
allowing a string with a unit is fine
this does not have a unit